### PR TITLE
add new board config for fmu-v3

### PR DESCRIPTION
**Pull request by kd0aij**
This uses a symbolic link to src/drivers/boards/px4fmu-v2 in src/drivers/boards/px4fmu-v3 to avoid the need to alter the cmake build machinery.

It may also be possible to reduce duplication in nuttx-configs/px4fmu-v3, but...

This leaves BOARD_NAME = PX4FMU_V2, making the fmu-v3 build target the same as v2 except with 2MB usable flash.

### CODE REVIEW

**Comment by LorenzMeier** [comment]
Symbolic links break on some OS, we've avoided them in the build system so far. Not sure if we should make an exception here, but I tend not to :(.

**Comment by kd0aij** [comment]
I thought it might be wise to make that a separate commit

File: src/drivers/boards/px4fmu-v3/board_config.h [hunk 1]
```diff
@@ -0,0 +1 @@
+../px4fmu-v2/board_config.h
```
Comment (LorenzMeier) [inline src/drivers/boards/px4fmu-v3/board_config.h]: Why can't you use that path here just as an include? I don't see the need for a symbolic link here.

Comment (kd0aij) [inline src/drivers/boards/px4fmu-v3/board_config.h]: now I understand; in order to avoid using a link

**Comment by dagar** [comment]
Even if OS support was fine, symlinks don't usually do well in version control.

**Comment by kd0aij** [comment]
now all I need is a Pixhawk with rev3 silicon :)

**Comment by pkocmoud** [comment]
@davids5 @kd0aij Try here: http://store.mrobotics.io/product-p/mro-pixhawk1-bb-mr.htm

**Comment by kd0aij** [comment]
Thanks, Phil. It's great to see that Jordi is still active in the DIY market.

**Comment by kd0aij** [comment]
I think this is ready for review and test. Looks good to me on an old Pixhawk1.

**Comment by davids5** [comment]
@kd0aij - FYI: I have a mro Pixhawk with rev 3 Silicon on it's way.

**Comment by kd0aij** [comment]
@davids5 Cool, now you'll have to do some real flight testing :)

**Comment by davids5** [comment]
@kd0aij 

As much as I can do with it sitting on the bench with 2 JTAGs connected LOL

**Pull request merged**